src/main/java/ch/ips/g2/applyalter/ApplyAlter.java

```diff
@@ -485,6 +485,14 @@ public void run() {
     }
 
     private void applySingleAlter(final Alter a, ApplyAlterExceptions aae) {
+        // There is no simple way to rollback auto committed command executions, so there is only SHARP mode allowed.
+        if (a.autocommit && !RunMode.SHARP.equals(getRunMode())) {
```

Comment:
This constraint seems to be overill, maybe we should only skip alter in DRY mode - other modes seems to be fine.